src/main.js

```diff
@@ -386,8 +386,8 @@ app.on('ready', function() {
     window_options = JSON.parse(fs.readFileSync(bounds_info_path, 'utf-8'));
   }
   catch (e) {
-    // follow Electron's defaults
-    window_options = {};
+    // follow Electron's defaults, except for window dimensions
+    window_options = {width: 1100, height: 800};
```

Comment:
As far as reading #354, the height should be smaller than 800 in order to avoid that the window is hidden by the taskbar.